packages/ui/src/components/editor/panels/Files/container/index.tsx

```diff
@@ -297,7 +297,12 @@ const FileBrowserContentPanel: React.FC<FileBrowserContentPanelProps> = (props)
   }
 
   const onSelect = (event, params: FileDataType) => {
-    if (params.type !== 'folder') {
+    if (params.isFolder) {
```

Comment:
I believe this is breaking the new functionality to double click into folders.

Reply:
this PR should make it so users can navigate into folders with a single-click

also updated to make it so double-clicks always navigate into folders